src/app/centers/createcenter/createcenter.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="centerForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input type="text" matInput required formControlName="name" />
+            <mat-error *ngIf="centerForm.controls.name.hasError('required')">
+              The Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Office</mat-label>
+            <mat-select required formControlName="officeId">
+              <mat-option *ngFor="let office of officeData ;let i = index" [value]="office.id" (click)="onclick(i)">
+                {{ office.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="centerForm.controls.officeId.hasError('required')">
+              Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Staff</mat-label>
+            <mat-select formControlName="staffId">
+              <mat-option *ngFor="let staff of staffData" [value]="staff.id">
+                {{ staff.displayName }}
+              </mat-option>
+            </mat-select>
+
+          </mat-form-field>
+
+          <table>
+            <tr>
+              <td>
+                <mat-checkbox [formControl]="disableSelect" (click)="disable()">Active</mat-checkbox>
+              </td>
+              <td [hidden]="!disableSelect.value">
+                <mat-form-field class="example-full">
+                  <mat-label>Activation On</mat-label>
+                  <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="activationDateDatePicker"
+                    formControlName="activationDate" required >
+                  <mat-datepicker-toggle matSuffix [for]="activationDateDatePicker"></mat-datepicker-toggle>
+                  <mat-datepicker #activationDateDatePicker></mat-datepicker>
+                  <mat-error *ngIf="centerForm.controls.activationDate.hasError('required')">
+                    Activation Date is <strong>required</strong>
+                  </mat-error>
+                </mat-form-field>
+              </td>
+            </tr>
+          </table>
+
+          <mat-form-field>
+            <mat-label>External Id </mat-label>
+            <input matInput type="number" formControlName="externalId">
+          </mat-form-field>
+
+
+          <mat-form-field class="example-full">
+            <mat-label>Submitted On</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDatePicker" formControlName="submittedOnDate"
+              >
```

Comment:
Please ensure there is no inconsistency with indentation like this anywhere in the code. Keep this on the same line.